byron/ledger/executable-spec/src/Ledger/UTxO.hs

```diff
@@ -8,9 +8,11 @@
 {-# LANGUAGE NamedFieldPuns             #-}
 {-# LANGUAGE RankNTypes                 #-}
 {-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE TemplateHaskell            #-}
 {-# LANGUAGE TypeFamilies               #-}
 {-# LANGUAGE UndecidableInstances       #-}
 
+{-# OPTIONS_GHC -fno-warn-orphans  #-}
```

Comment:
Why do we need this?

Reply:
Turns out - we don't! Not sure why I had it in there, thanks for checking.